src/etc/rc.freebsd

```diff
@@ -41,6 +41,7 @@ rc_enabled()
 {
 	rc_filename=${1}
 	name=${2}
+	unset rcvar
```

Comment:
understood, small request though: can we use this idiom instead?

rcvar=

Reply:
yes, that looks reasonable - I'm not going to be able to (easily) test it anymore though...

Reply:
I‘ll use your commit but edit as discussed. Will do testing, although mostly for regressions which are not expected either way :)